pkg/admissionpolicy/mutate_test.go

```diff
@@ -568,7 +570,25 @@ func Test_MutateResource(t *testing.T) {
 			resource, err := kubeutils.BytesToUnstructured(tt.rawResource)
 			assert.NilError(t, err)
 
-			response, err := mutateResource(policy, *resource)
+			gvk := resource.GroupVersionKind()
+			gvr := schema.GroupVersionResource{
+				Group:    gvk.Group,
+				Version:  gvk.Version,
+				Resource: strings.ToLower(gvk.Kind) + "s",
+			}
```

Comment:
We can use the REST Mapper:
https://github.com/kyverno/kyverno/blob/0ec1222ef0cbb94e8e738bd32ef4916d1191938b/pkg/admissionpolicy/validate_test.go#L196